db_service/setup_db.go

```diff
@@ -27,29 +27,48 @@ import (
 	"code.cloudfoundry.org/lager"
 	"github.com/go-sql-driver/mysql"
 	"github.com/jinzhu/gorm"
+	"github.com/spf13/viper"
 )
 
+const (
+	caCertProp     = "db.ca.cert"
```

Comment:
This approach seems inconsistent with serve.go. I'm not sure what the benefit is of adding a layer of indirection as opposed to using the strings in-line.

Reply:
These are used twice, so I wanted to avoid a double hard-coded variable in case one got changed. What do you think about pulling these out into a separate config package?

Something that would hold string constants and set up the legacy mappings? I always feel a little bit iffy about that as a design pattern because I feel like it lets access creep deep into the code which means people start panicing rather than returning errors which I feel is an anti-pattern. But maybe it can't be helped in this case.

Thoughts?

Reply:
I could see the benefit of either approach. My primary appeal is for consistency across the code base. As you're closer to this code, I'll defer to your best judgement :)